lib/components/artist/related_artists/index.js

```diff
@@ -18,20 +18,34 @@ class RelatedArtists extends React.Component {
   }
 
   layoutState() {
-    const window = Dimensions.get('window');
-    const isPad = window.width > 700;
-    const isPadHorizontal = window.width > 1000;
+    const width = Dimensions.get('window').width;
+    const isPad = width > 700;
+    const isPadHorizontal = width > 1000;
 
-    const columnCount = (isPad ? (isPadHorizontal ? 4 : 3) : 2);
+    // TODO: Document what these margins are based on.
```

Comment:
@sarahscott @mennenia Would love to have this sorted. They are really magic numbers right now. I’d also prefer to distinguish between side-margins and in-between margins and then calculate the column width based on that.